services/dispatcher/dispatcher.cpp

```diff
@@ -92,90 +89,96 @@ namespace services::dispatcher {
         load_result_.clear();
     }
 
-    void dispatcher_t::load_from_wal_result(components::session::session_id_t& session, std::vector<services::wal::record_t> &records) {
-        load_count_answers_ = records.size();
+    void dispatcher_t::load_from_wal_result(components::session::session_id_t& session, std::vector<services::wal::record_t> in_records) {
+        // TODO think what to do with records
+        records_ = std::move(in_records);
```

Comment:
@kotbegemot @makeyavelly plz pay attention to this. We decided to add this with @makeyavelly because before we got it as non const reference
`void dispatcher_t::load_from_wal_result(components::session::session_id_t& session, std::vector<services::wal::record_t>& in_records)`
This potentially can lead to danling references.